lib/renderer.coffee

```diff
@@ -9,7 +9,7 @@ pandocHelper = null # Defer until used
 markdownIt = null # Defer until used
 {scopeForFenceName} = require './extension-helper'
 mathjaxHelper = require './mathjax-helper'
-pathWatcher = require 'pathwatcher-without-runas'
+pathWatcher = require atom.packages.resourcePath + '/node_modules/pathwatcher/lib/main'
```

Comment:
okay, will try it later on.
But as we are using the global pathwatcher, we really have to check,
that we destroy pathwatchers as soon as a preview is closed, otherwise there could be a pathwatcher "leakage".